Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Dashboard/Customer.hs

```diff
@@ -74,7 +77,7 @@ blockCustomer merchantShortId customerId = do
   unless (merchant.id == merchantId) $ throwError (PersonDoesNotExist personId.getId)
 
   runTransaction $ do
```

Comment:
bro, can you add deletion of auth token from registration_token table as well in this ?